packages/e2e-tests/tests/earlyWarning/earlyWarning.spec.ts

```diff
@@ -0,0 +1,704 @@
+import { expect, test } from '@playwright/test';
+import { formatNumber } from '#utils/common';
+import fixtureData from './earlywarning.json';
+test.use({ storageState: 'playwright/.auth/user.json' });
+test.describe('Field Report', () => {
```

Comment:
Let's describe the test in detail. For example
```suggestion
test.describe('Early Warning Field Report', () => {
```